### PR TITLE
Meow Voulges

### PR DESCRIPTION
**Pull request by Meowkov**
Added Ancient Long Voulge
Added Fabulous Voulge

### CODE REVIEW

**Review by Meowkov — COMMENTED** [review]
Looking good, approved.

**Review by Yeldur — APPROVED** [review]
Approving on Meow's behalf

**Review by Erdertus — DISMISSED** [review]
BvS with reduction in cut damage to balance the tier - visual consistency with the rest of the weapons.

You can dismiss this change if you want though.

**Comment by Yeldur** [comment]
> BvS with reduction in cut damage to balance the tier - visual consistency with the rest of the weapons.
> 
> You can dismiss this change if you want though.

You be the boss of poles, I shall update them like so milord - Will resubmit once done 🫡

**Review by Erdertus — APPROVED** [review]

File: items.json [hunk 1]
```diff
@@ -38467,7 +38631,7 @@
     "weight": 1.0,
     "tier": 1.313482,
     "rank": 2,
-    "requirement": 0,
+    "requirement": 3,
```
Comment (Muparadzi) [inline items.json:38634]: needs removing

**Review by Muparadzi — CHANGES_REQUESTED** [review]

File: items.json (hunk 1 above)
Comment (Meowkov) [reply items.json]: done :D

**Review by Muparadzi — APPROVED** [review]

**Review by Yeldur — APPROVED** [review]
After double checking things with Meow and testing myself, I'm happy - Approving

**Pull request merged**